### ISSUE TITLE
Design a Survey for Gathering Information on Open Source Organizations and Projects

### ISSUE THREAD

**Issue by reginankenchor** [sustainers/academic-map#101]
**Description:**
Design a comprehensive survey aimed at collecting information that will help users understand academic OSS. The goal is to gather insights into what specific data people want to know when they look at an open-source project or organization, including roles, contributions, project impact, collaboration opportunities, and key contacts.

**Key Objectives for the Survey:**

- Identify the types of information that users find most valuable (e.g., personnel, roles, funding sources, technical platforms, community engagement).
- Understand how users would prefer to access and organize this information (e.g., searchable directories, visual maps, project timelines).
- Gather insights into which roles or departments in an organization users are most interested in.
- Learn what collaboration or engagement opportunities users want to explore with open-source organizations and projects.
- Identify additional features or challenges users face when navigating the open-source ecosystem.

**Expected Outcomes:**

- A data-driven survey that focuses on understanding the needs and preferences of the community regarding open source projects and organizations.
- Clarity on what information is most crucial for building a resource that highlights "who is doing what" within an organization or project.
- Improved design and content focus for the organization section, which will be extended to other sections for further improvements.

**Action Items:**
- Create a set of questions based on the objectives above.
- Review and finalize the survey with feedback from @RichardLitt 
- Prepare to distribute the survey to relevant open-source communities and academic partners.

**Comment by RichardLitt** [comment]
>  Design a comprehensive survey aimed at collecting information that will help users understand the landscape of open source organizations and projects. The goal is to gather insights into what specific data people want to know when they look at an open-source project or organization, including roles, contributions, project impact, collaboration opportunities, and key contacts.

A key difference: this isn't the landscape of open source orgs and projects, but explicitly academic OSS. 

Also, may be worth looking at this: https://investinopen.org/blog/nnouncing-150-000-award-to-ioi-to-advance-understanding-of-the-research-software-ecosystem/. I think their project is slightly different - it's more project focused.

**Comment by reginankenchor** [comment]
> > Design a comprehensive survey aimed at collecting information that will help users understand the landscape of open source organizations and projects. The goal is to gather insights into what specific data people want to know when they look at an open-source project or organization, including roles, contributions, project impact, collaboration opportunities, and key contacts.
> 
> A key difference: this isn't the landscape of open source orgs and projects, but explicitly academic OSS.
> 
> Also, may be worth looking at this: https://investinopen.org/blog/nnouncing-150-000-award-to-ioi-to-advance-understanding-of-the-research-software-ecosystem/. I think their project is slightly different - it's more project focused.

Thank you, I’ve made adjustments to the description. I reviewed the IOI project, and you're right; the two projects are slightly different. Our project adopts a broader approach by examining various entities in academia that work with open-source software—such as organizations, universities, service providers, and funders—offering a more comprehensive map of the open-source ecosystem in academia. In contrast, IOI has a narrower focus, concentrating on research software development and the infrastructure that supports it. Their work emphasizes identifying existing connections, gaps, and vulnerabilities within the research software ecosystem, aiming to improve its sustainability and resilience. Our data collection methods also vary, but we could certainly draw inspiration from IOI’s approach, particularly in identifying key gaps and opportunities. This could help us align better with our goal of creating a resource for navigating the academic open-source ecosystem.

**Comment by RichardLitt** [comment]
If we want, we can also connect with them directly. I've had a few people from IOI over the years on the SustainOSS podcast, and know a few of the excellent people on the IOI team.

**Comment by reginankenchor** [comment]
> If we want, we can also connect with them directly. I've had a few people from IOI over the years on the SustainOSS podcast, and know a few of the excellent people on the IOI team.

That sounds like a great idea! I think connecting with them would be valuable for our work.

**Comment by RichardLitt** [comment]
Cool. Let's wait until we have a clearer goal and plan for how this would look. :)

**Comment by reginankenchor** [comment]
> Cool. Let's wait until we have a clearer goal and plan for how this would look. :)

Okay.

**Comment by reginankenchor** [comment]
Hi @RichardLitt, I have added you as a collaborator on the first draft of the survey. Please review it and drop your comments here. Are there any questions you think should be included, removed, or modified?
Thank you.

**Comment by RichardLitt** [comment]
Thanks @reginankenchor. Where did you add me?

**Comment by RichardLitt** [comment]
Ok, looked over it, and made some edits: 
- I trimmed down the intro - it had a lot of content in it, and I think it could be a bit easier to parse;
- I changed "open-source" to "open source" across the document. https://opensource.org/blog/is-open-source-ever-hyphenated
- I've added a few different options to some of the questions. 

This survey is a bit long - and I'm not sure how useful it would be. I'm not sure how we would use a data. Do you have a plan for how to interpret the responses and to use them to add more content to the map?

**Comment by reginankenchor** [comment]
> Ok, looked over it, and made some edits:
> 
>     * I trimmed down the intro - it had a lot of content in it, and I think it could be a bit easier to parse;
> 
>     * I changed "open-source" to "open source" across the document. https://opensource.org/blog/is-open-source-ever-hyphenated
> 
>     * I've added a few different options to some of the questions.
> 
> 
> This survey is a bit long - and I'm not sure how useful it would be. I'm not sure how we would use a data. Do you have a plan for how to interpret the responses and to use them to add more content to the map?

Thank you, @RichardLitt . Yes, I’ll review the questions and see what we can trim down. My goal is to interpret the responses and identify any missing information to add to our resource.

**Comment by RichardLitt** [comment]
I don't think that this survey would highlight many more _resources_, although it may certainly highlight some values people have around what they look for. I think if we're just looking for organizations or foundations, then this should be more targeted at projects people work on, not just what they're looking for. 

I'm not sure the academic map should have just projects on it - for instance, dependencies of R - if they're not connected to a foundation, organization, university, or some other entity outside of themselves. Otherwise, we'd just be including all of the dependency tree of science. Do you follow me here?

**Comment by reginankenchor** [comment]
> I don't think that this survey would highlight many more _resources_, although it may certainly highlight some values people have around what they look for. I think if we're just looking for organizations or foundations, then this should be more targeted at projects people work on, not just what they're looking for.
> 
> I'm not sure the academic map should have just projects on it - for instance, dependencies of R - if they're not connected to a foundation, organization, university, or some other entity outside of themselves. Otherwise, we'd just be including all of the dependency tree of science. Do you follow me here?

Yes, I do. What are you proposing?

**Comment by RichardLitt** [comment]
I think, if we have a survey, it should be based on two things - **one**, which foundations, orgs, or universities does the responder think related to open source in academia? and **two**, what would be useful for you to know when looking for other such types of organizations? What would you want listed? 

I think separating out these into two sections would help clarify that.

**Comment by reginankenchor** [comment]
> I think, if we have a survey, it should be based on two things - **one**, which foundations, orgs, or universities does the responder think related to open source in academia? and **two**, what would be useful for you to know when looking for other such types of organizations? What would you want listed?
> 
> I think separating out these into two sections would help clarify that.

I agree with your suggestions. I will adjust the survey to include these two sections. The first question will focus on identifying relevant organizations, while the second question will address information needs.

**Comment by reginankenchor** [comment]
Hi @RichardLitt ,
I have made adjustments to the survey questions, incorporating feedback from both you and @Nolski. Could you kindly review the updated questions and let me know if any further adjustments are needed?  [Survey Link](https://docs.google.com/forms/d/1pdcKzLiS_DfeoZFEq6bFMzIt0I7sWLyTRHABr_m8hlI/edit)

**Comment by RichardLitt** [comment]
I think this looks great. Let's post it. :) 

What is your plan for sending it out?

**Comment by reginankenchor** [comment]
> I think this looks great. Let's post it. :)
> 
> What is your plan for sending it out?
Hi @RichardLitt,
I plan to start distributing the survey on November 20th. The rollout will begin on my social platforms, followed by sharing it with my network of friends in various open source communities and the communities I am part of. We can then reshare it every two days for the next two weeks, with the survey closing on December 3rd.
This means the survey will run from November 20th to December 3rd.

**Comment by reginankenchor** [comment]
Mapping the Open Source Ecosystem in Academia survey officially closed on March 20th, with a total of 22 responses. I’m now working on a blog post based on the insights gathered, and a first draft should be available by April 2nd. @RichardLitt

**Comment by reginankenchor** [comment]
Hi @RichardLitt I’ve put together a draft of the [survey report](https://docs.google.com/document/d/1IzSJbX0DF3ZHazXXTyWzIHCa-DmhQ7c-f9ldDA4FlVk/edit?usp=sharing) for your review. I’m still finalizing the References section, but the rest is ready for feedback.

**Comment by reginankenchor** [comment]
> Hey [@reginankenchor](https://github.com/reginankenchor). This looks great! I'm curious – do you have an expectation for where this report will go? Are you planning on publishing it on the SustainOSS website, or the map, or in a journal or something?

@RichardLitt Thank you! Yes, I'm planning to publish first on either the SustainOSS website or the map. I'll follow your advice.

**Comment by RichardLitt** [comment]
Would you be interested in publishing it more widely, perhaps at a conference? :) Might be able to turn this into such a thing, if that's of interest.

**Comment by RichardLitt** [comment]
I would call this done. I've merged #198, which has the report in it. Thank you, @reginankenchor!
